.github/workflows/auto_triage.yml

```diff
@@ -11,7 +11,30 @@ jobs:
     permissions:
       issues: write
     steps:
-      - run: gh issue edit "$NUMBER" --add-label "$LABELS"
+      - name: Check if issue needs triage
+        id: check-labels
+        run: |
+          # Get current labels and issue type - if any exist, the issue has been looked at
+          label_count=$(gh issue view "$NUMBER" --json labels --jq '.labels | length')
```

Comment:
Don't issues have a default `TRIAGE` label when created? If so, that means the bot will always skip.

Reply:
Not exactly. For us maintainers who have the ability to see a label on an issue, the issue template defaults to adding the TRIAGE label.

<img width="945" height="529" alt="image" src="https://github.com/user-attachments/assets/07718fe2-cb3f-4954-89aa-d337b66aaba2" />

There's a way to manually remove it before creating the issue but I don't think it hurts improving the bot a bit (specially if we then copy / paste its implementation to other repos).
<img width="790" height="787" alt="image" src="https://github.com/user-attachments/assets/5770c7d4-f1f7-4f66-8467-ebf2fbe5208a" />

Reply:
OK, I understand you now. It still works.

- If the automatic `TRIAGE` label is added by a maintainer, the bot will skip.
- If a maintainer removes the label but adds an issue type, the bot will also skip. That way, you won't need to manually remove the `TRIAGE` label if you've already added an issue type while creating the issue.